packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/configure-packages/react-native/installsdk.md

```diff
@@ -0,0 +1,53 @@
+### Install npm package
+
+```bash
+$ npm install @okta/okta-react-native --save
+$ react-native link @okta/okta-react-native
+```
+
+### Install iOS OIDC SDK
+
+1. [**CocoaPods**](https://guides.cocoapods.org/using/getting-started.html)
+
+***React Native >= 0.60***: With React Native 0.60 pods are added to podfile automatically. Run `pod install` command to install dependecies:
+```
+cd ios
+pod install
+```
+***React Native < 0.60***: Make sure your `Podfile` looks like this:
```

Comment:
added "that"

***React Native < 0.60***: Make sure that your `Podfile` looks like this: